addons/mail/static/src/composer/emoji_picker.js

```diff
@@ -64,6 +66,11 @@ export class EmojiPicker extends Component {
         });
     }
 
+    afterLoadEmoji() {
```

Comment:
This is interresting ! I'm quite curious as to whether or not it's a good practice to create an empty function to facilitate testing.
In my opinion, it is the tests that should adapt to the code and not the opposite.

An alternative I have in mind is to patch `setup()`, call the `super` in it and call a lifecycle hook that comes after the `onWillStart` to make the defer

Reply:
in this case, i would personally just load emoji data manually in the test itself

Reply:
change done (manually `loadEmojiData()` in test)